testing/run_tests.py

```diff
@@ -281,36 +281,105 @@ def RunDartTest(build_dir, test_packages, dart_file, verbose_dart_snapshot, mult
                       forbidden_output=forbidden_output, expect_failure=expect_failure)
 
 
-def EnsureDebugUnoptSkyPackagesAreBuilt():
+def EnsureDebugUnoptSkyPackagesAreBuilt(check_only):
```

Comment:
nit: here and elsewhere, change `check_only` to something like `do_build`.

It's slightly confusing right now to track how this gets set as an inversion of a top level argument to the script.